ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -343,8 +338,9 @@ Open Get Started Sidebar And Return Status
     [Arguments]  ${card_locator}
     Wait Until Element Is Visible    xpath:${card_locator}
     Wait Until Element Is Enabled     xpath:${card_locator}    timeout=20s     error=Element is not clickbale  #robocop : disable
-    Click Element    ${card_locator}
-        ${status}=  Run Keyword and Return Status  Wait Until Page Contains Element    xpath://div[contains(@class,'pf-v5-c-drawer__panel-main')]
+    Mouse Down    ${card_locator}
+    Mouse Up    ${card_locator}
```

Comment:
why?

Reply:
Because of clicking element fails on "Other element would receive the click", no matter how I try to optimize the Xpath. For example:
`Click Element    //*[contains(@data-testid,"cardtitle") and contains(text(),'Anaconda Professional')]/../../..//input[@type='radio']`

> robot.errors.HandlerExecutionFailed: ElementClickInterceptedException: Message: element click intercepted: Element <input aria-labelledby="anaconda-ce" id="anaconda-ce-selectable-card-id" name="odh-explore-selectable-card" class="pf-v5-c-radio__input" type="radio" aria-invalid="false" data-ouia-component-type="PF5/Radio" data-ouia-safe="true" data-ouia-component-id="OUIA-Generated-Radio-1"> is not clickable at point (613, 240). Other element would receive the click: <label class="pf-v5-c-radio__label" for="anaconda-ce-selectable-card-id"></label>
>   (Session info: chrome=122.0.6261.69)

Reply:
the card should be clickable yes. In the past they disabled this option, but iirc they put it back. This would need a double check

Reply:
That is what I do :) click on the whole card:
![image](https://github.com/red-hat-data-services/ods-ci/assets/9913945/f83725bc-1958-4fcb-bd4e-400a3db714d0)

Reply:
yeah but why not using Click Element instead of Mouse Down/Up?

Reply:
@bdattoma yes, they have put it back already

@manosnoam I don't think so - you click on the child div element instead of on the whole card:

![Screenshot from 2024-04-08 10-38-26](https://github.com/red-hat-data-services/ods-ci/assets/12250881/1fc31c20-a80e-4586-a14f-67d890b9b560)

Reply:
I thought so too, that clicking on `//*[contains(@data-testid,"cardtitle") and contains(text(),'Anaconda Professional')]` will work, but as I explained, this fails in our tests. 
In "Browser Developer mode" you can find the element, but in Selenium runtime, it fails to click on it with error `Other element would receive the click`. Try to debug test `Verify Anaconda Professional Is Available In RHODS Dashboard Explore/Enabled Page` and see...